bayesian/utils.py

```diff
@@ -381,11 +381,20 @@ def get_analyses_for_snyk(self):
 
             if "cve" in result_data[0]:
                 # if cve if present
+                rec_version = result_data[0].get('package', {}).get(
+                                                'latest_non_cve_version', {})
                 clubbed_data.append({
-                    "recommended_versions": result_data[0]['cve']['sfixed_in'],
-                    "link": self.get_link(),
+                    "recommended_versions": rec_version,
+                    "snyk_pkg_link": self.get_link(),
                 })
+                logger.info("latest non cve version for {eco} {pkg} is {ver}".format(
+                    eco=self.ecosystem,
+                    pkg=self.package,
+                    ver=rec_version
+                ))
             else:
+                logger.info("Unable to find recommended version for Package Object:\n{}\n".format(
```

Comment:
@deepak1725 This else logger is printing incorrect msg. The else part is for CVE. so basically whenever there is no cve, it will print this msg. remove it.

Reply:
Ack!